testutils/generators.go

```diff
@@ -361,9 +362,11 @@ func RandNumberOverRange(min int64, max int64) int64 {
 
 // GenerateRandBytes generates random bytes array
 func GenerateRandBytes() []byte {
-	var hash []byte
-	fuzz.New().NilChance(0).Fuzz(&hash)
-	return hash
+	u, err := uuid.NewV4()
+	if err != nil {
+		panic("can't create uuid:" + err.Error())
```

Comment:
is error even possible? forcing panic in testutils is too risky, will cause all test run to fail once happened. I'd suggest to ignore error or to handle it another way